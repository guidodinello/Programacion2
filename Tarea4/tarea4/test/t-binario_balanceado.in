# Prueba de tiempo binarioBalanceado
tiempo_binarioBalanceado 10000000 1 3
Fin
