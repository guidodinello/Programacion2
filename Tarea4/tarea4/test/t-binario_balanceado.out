1>#  Prueba de tiempo binarioBalanceado.
2>
 Construyendo el arreglo. 
 Construyendo el Ã¡rbol. 
3>Fin.
